patroni/postgresql/__init__.py

```diff
@@ -277,7 +278,7 @@ def pg_isready(self) -> str:
 
         :returns: 'ok' if PostgreSQL is up, 'reject' if starting up, 'no_resopnse' if not up."""
 
-        r = self.config.local_connect_kwargs
+        r = self.connection_pool.conn_kwargs
         cmd = [self.pgcommand('pg_isready'), '-p', r['port'], '-d', self._database]
```

Comment:
```suggestion
        connect_kwargs = self.connection_pool.conn_kwargs
        cmd = [self.pgcommand('pg_isready'), '-p', connect_kwargs['port'], '-d', self._database]
```

Reply:
And the other references of this var in this method

Reply:
This will be unrelated change, will bloat PR and make it harder to review.